doc/src/core-transmutation.md

```diff
@@ -0,0 +1,143 @@
+Challenge: Preconditions for core transmuting methods
+
+## Status
+
+\[Open | Resolved | Expired\]: Open
+
+## Dates
+
+Start date:
+
+End date:
+
+## Goal
+
+Confirm the correctness of value transmutations performed by libcore, including those transmutations exposed as public methods provided by libcore.
```

Comment:
What does "correctness" mean here?

Reply:
(I don't mind if we change the text to say "soundness" rather than "correctness". I think the reason I wrote "correctness" is that I was thinking there may be cases where the soundness of some caller may require stronger correctness conditions on transmute itself.)